tests/code.spec.ts

```diff
@@ -258,7 +258,7 @@ test('press backspace after code block can jump into start of code block', async
   expect(index).toBe(0);
 });
 
-test('press ArrowUp after code block can jump into start of code block', async ({
+test.only('press ArrowUp after code block can select code block', async ({
```

Comment:
Please remove `test.only`, and `IfSelectRectExist` -> `selectedRectsExist`, it's always a good habit to pay attention to word properties and plural cases 👻

Reply:
Thanks for reminding!!!